test-cases/features/gce-multiple-dc-shutdown-30mins.yaml

```diff
@@ -1,7 +1,7 @@
 # TODO: need to qualify
 test_duration: 45
 
-stress_cmd: cassandra-stress write no-warmup cl=QUORUM duration=30m -schema 'replication(factor=1)' -mode cql3 native -rate threads=100 -pop seq=1..10000000 -log interval=5
+stress_cmd: cassandra-stress write no-warmup cl=LOCAL_QUORUM duration=30m -schema 'replication(factor=1)' -mode cql3 native -rate threads=100 -pop seq=1..10000000 -log interval=5
 
 n_db_nodes: 2
```

Comment:
Just 2 nodes? In one region?

I don't see this file to be used anywhere. Maybe better to delete it instead of fixing dead code?

Reply:
yes, I also did not find using of this. I see that it was created by @fruch and changed by you one year ago ("improvement(c-s): remove redundant jmx port specification ").
@fruch dow we need it? 
If yes - it should be fixed

Reply:
If it's not mentioned in any pipeline, yes we can remove it.

It was ported by me from the avocado configuration files, if since then it wasn't used, no one needs it.